ts/features/lollipop/saga/index.ts

```diff
@@ -28,6 +41,10 @@ export function* generateLollipopKeySaga() {
       // a public key tied with it.
       const publicKey = yield* call(getPublicKey, maybeOldKeyTag.value);
       yield* put(lollipopSetPublicKey({ publicKey }));
+      const mixPanelEnabled = yield* select(isMixpanelEnabled);
+      if (mixPanelEnabled) {
+        yield* call(trackLollipopKeyGenerationSuccess, publicKey.kty);
+      }
```

Comment:
Looking better I guess we even don't need to track this event. Here we are collecting an already generated key only to be sure it already exists.

Reply:
If we do not track this event, there is a discrepancy with the behaviour of the old code (i.e., the old code was tracking the KeyGenerationInfo of a successfully generated key). Do you wish to remove this behaviour?

Reply:
Yep, but in this specific point we are _getting_ the key and not _generating_ it:
`const publicKey = yield* call(getPublicKey, maybeOldKeyTag.value);`. Or I miss something? 🤔

Reply:
If you look at the previous code, in the startup saga, KeyGenerationInfo were sent regardless of them being a failure of a success 
```
367  // Track crypto key generation info
368  if (O.isSome(keyTag)) {
369    yield* call(trackMixpanelCryptoKeyPairEvents, keyTag.value);
370  }
``` 

so in this PR, that event has been moved here. We can drop it but then the code behaviour is changed

Reply:
Okay I get what you mean and It's right. But now we only need to track it when the key is actually generated. We can drop this one.